build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext {
         // if you change this version remember to change the version in the plugins closure
-        springBootVersion = '2.1.1.RELEASE'
+        springBootVersion = '2.1.4.RELEASE'
```

Comment:
Do we also need to change this in the plugins closure?

Reply:
If this is the case and they should always be the same, could we create some constant to refer to for this value?

Reply:
I have already tried before to make a constant for this.  Unfortunately so far I have not been able to.  The buildscript part is evaluated differently than the plugins portion.